### ISSUE TITLE
Highlighted user research links in header fail a11y contrast settings

### ISSUE THREAD

**Issue by pcraig3** [alphagov/digitalmarketplace-frontend-toolkit#473]
Hello!

Should be easy to fix: when you focus on a link in the [user research banner](https://alphagov.github.io/digitalmarketplace-frontend-toolkit/user-research-consent-banner.html), the white text on yellow background fails an a11y contrast check.

<img width="1410" alt="Screen Shot 2019-07-03 at 4 01 51 PM" src="https://user-images.githubusercontent.com/2454380/60621704-5b1be000-9dac-11e9-98bf-89f2a62f83ba.png">

Easy possible fix: black link text when focused.

**Comment by katstevens** [comment]
@pcraig3 Thanks Paul! We're looking at accessibility issues in the upcoming quarter so will add this to our list.
